File: pwa-benchmark/src/Components/Item/Item.js
import React from 'react';
import LoremIpsum from '../../utils/loremipsum';

import './Item.css';

function PictureItem(props) {
    const file = require(`./../../utils/pictures/picture${props.index}.jpg`);
    return (
        <div className='item picture'>
            <img src={file} alt="Picture" />
        </div>
    );
}

function TextItem(props) {
    return (<p className="item text">{LoremIpsum[props.index]}</p>);
}

export { PictureItem, TextItem }

File: Android/Benchmark/app/src/main/java/com/example/benchmark/model/Item.java
package com.example.benchmark.model;

public class Item {
    public String mData;
    //1 for text, 2 for pictures
    public int mType;

    public Item(String data, boolean picture) {
        mData = data;
        mType = picture ? 2 : 1;
    }
}
